src/crypto.js

```diff
@@ -40,12 +40,10 @@ exports.getRandomValues = function (buffer) {
     view = new Uint8Array(buffer),
     i;
 
-  if (offset + size > buf.length) {
-    throw new Error("Insufficient Randomness Allocated.");
-  }
-
+  // Hack: just wrap around.
```

Comment:
Ugh, too much editing.  I meant add hidden state -- the amount of entropy that this component believes that it needs.